docs/outputs.rst

```diff
@@ -0,0 +1,731 @@
+.. include:: links.rst
+
+.. _outputs:
+
+---------------------------
+Outputs of *fMRIPost-AROMA*
```

Comment:
Actually, it looks like there are no specific mentions of ICA-AROMA outputs at this point. Maybe let's just remove this file until we have something to say?

Reply:
I just drafted a minimal version of the AROMA output documentation. It should be largely accurate now.